recipe/meta.yaml

```
@@ -23,34 +23,35 @@ requirements:
     - python >=3.8,<3.11
     - pip
     - coiled
-    - numpy ==1.21.6
-    - pandas ==1.4.2
+    - numpy ==1.23.3
+    - pandas ==1.5.1
     - dask =={{ dask_version }}
     - distributed =={{ distributed_version }}
-    - fsspec ==2022.5.0
-    - s3fs ==2022.5.0
-    - gcsfs ==2022.5.0
-    - pyarrow ==8.0.0
-    - jupyterlab ==3.4.2
-    - dask-labextension ==5.2.0
-    - lz4 ==4.0.0
-    - ipywidgets ==7.7.0
+    - fsspec ==2022.10.0
+    - s3fs ==2022.10.0
+    - gcsfs ==2022.10.0
+    - pyarrow ==9.0.0
+    - jupyterlab ==3.4.8
+    - dask-labextension ==5.3.0
+    - lz4 ==4.0.2
+    - ipywidgets ==8.0.2
```

```suggestion
    - ipywidgets ==7.7.2
```
Coiled requires <8.0: https://github.com/conda-forge/coiled-feedstock/blob/main/recipe/meta.yaml